tests/test_asset.py

```diff
@@ -80,27 +79,20 @@ def test_asset_update_metadata(asset_mock):
     assert updated_info["tags"] == ["othertag1", "othertag2"]
 
 
-def test_asset_get_download_url(assets_fixture):
```

Comment:
why are we removing this test?

Reply:
1. It's not a public method
2. It is used by asset::download (covered by tests) and asset::get_stac_asset_url (not covered anyway)